jsfuck.js

```diff
@@ -76,16 +76,16 @@
     'Y':   null,
     'Z':   null,
 
-    ' ':   '(NaN+[]["flat"])[11]',
+    ' ':   '(NaN+[]["at"])[11]',
     '!':   null,
     '"':   '("")["fontcolor"]()[12]',
     '#':   null,
     '$':   null,
-    '%':   'Function("return escape")()([]["flat"])[21]',
+    '%':   'Function("return escape")()([]["at"])[19]',
     '&':   '("")["fontcolor"](")[13]',
     '\'':  null,
-    '(':   '([]["flat"]+"")[13]',
-    ')':   '([0]+false+[]["flat"])[20]',
+    '(':   '([]["at"]+"")[11]',
+    ')':   '([0]+false+[]["at"])[18]',
```

Comment:
Same here: `(""+[]["at"])[12]`

Reply:
It seems `""+` and `+""` has a little difference in characters produced. `([]["at"]+"")[12]` is actually 3 characters shorter.